source/user-manual/manager/manual-integration.rst

```diff
@@ -43,6 +45,21 @@ b. For SysV Init:
 
   # service wazuh-manager restart
 
+
+Optional filters
+^^^^^^^^^^^^^^^^
+
+The *optional filters* fields are used by ``Integrator`` to apply certain filters to the alerts being processed to determine which ones should be sent to the external platforms. Only the alerts that meet the filter conditions are sent. If no filters are specified, all alerts are sent.
+
+The following considerations must be taken into account when the filters are set:
+   
+   - It is possible to specify multiple group names using the ``<group>`` field with a comma-separated list. If the alert's group matches any of the groups in the list, the alert is sent. Otherwise, it is ignored.
+   - It is possible to specify multiple rule ids using the ``<rule_id>`` field with a comma-separated list. If the alert's rule id matches any of the ids in the list, the alert is sent. Otherwise, it is ignored.
```

Comment:
```suggestion
   - It is possible to specify multiple rule IDs using the ``<rule_id>`` field with a comma-separated list. If the alert's rule ID matches any of the IDs in the list, the alert is sent. Otherwise, it is ignored.
```

Reply:
Done